core/Cargo.toml

```diff
@@ -350,6 +350,7 @@ prometheus-client = { version = "0.22.2", optional = true }
 tracing = { version = "0.1", optional = true }
 # for layers-dtrace
 probe = { version = "0.5.1", optional = true }
+crc32c = "0.6.5"
```

Comment:
This dependency doesn't seem to be grouped cleanly like others.

Reply:
Nice catch, would like to send a PR to fix this?